src/lib/y2network/widgets/vlan_id.rb

```diff
@@ -52,6 +55,25 @@ def minimum
       def maximum
         9999
       end
+
+    private
+
+      def modified?
+        @config.vlan_id != value
+      end
+
+      def suggested_name
+        "vlan#{value}"
+      end
+
+      def suggest_vlan_name
+        Yast::Popup.YesNo(
+          format(
+            _("Would you like to adapt the interface name from '%s' to '%s'?"),
```

Comment:
I'd add some more noise why the change is being suggested. Something like "VLAN with id  20 is being defined. Would you mind to modify device name from vlan0 to vlan20"

Reply:
![Screenshot from 2021-01-25 08-27-58](https://user-images.githubusercontent.com/7056681/105680207-51565200-5ee7-11eb-883c-61b4da049354.png)